src/Kunstmaan/ConfigBundle/Controller/ConfigController.php

```diff
@@ -73,15 +68,23 @@ public function __construct(
         AuthorizationCheckerInterface $authorizationChecker,
         EntityManagerInterface $em,
         array $configuration,
-        ContainerInterface $container,
-        FormFactoryInterface $formFactory
+        /* ContainerInterface $container, */
+        /* FormFactoryInterface */ $formFactory
     ) {
         $this->router = $router;
         $this->templating = $templating;
         $this->authorizationChecker = $authorizationChecker;
         $this->em = $em;
         $this->configuration = $configuration;
-        $this->container = $container;
+
+        if (func_num_args() > 6) {
+            @trigger_error(sprintf('Passing the "container" as the sixth argument in "%s" is deprecated in KunstmaanConfigBundle 5.1 and will be removed in KunstmaanConfigBundle 6.0. Remove the "container" argument from your service definition.', __METHOD__), E_USER_DEPRECATED);
+
+            $this->formFactory = func_get_arg(6);
+
```

Comment:
The services.yml file needs to be updated to.

Reply:
Good catch! Fixed!

Reply:
And I also think we need to keep the container variable set. Because when people are still using the container, they will still get an error because the private $container does not exist anymore.

Reply:
The container propery was private and there is no getter defined in the controller, so extending the controller will have the container property available in user code. So I think we can safely remove the property

Reply:
True !